### PR TITLE
Adjustment for current token handling behavior #718

### LINKED ISSUE #718: Adjust tests to the new token handling behavior
We have several session token tests failing after https://github.com/nspcc-dev/neofs-node/pull/2731, the node is fine, tests need to be adjusted.

https://http.fs.neo.org/HXSaMJXk2g8C14ht8HSi7BBaiYZ1HeWh2xnWPGQCg4H6/1230-1706624712/index.html#suites/0598eb478b736df2d02c5aa8b49c27f4

### PR DESCRIPTION
**Pull request by MaxGelbakhiani**
Fixes #718

### CODE REVIEW

**Comment by evgeniiz321** [comment]
when this behavior was introduced? We should use the same version for our new tests (currently we use version: 'tags/v0.39.2', see in .github) and update them as well - dynamic_env_pytest_tests/tests/session_token/test_static_session_token_container.py

**Comment by MaxGelbakhiani** [comment]
> when this behavior was introduced? We should use the same version for our new tests (currently we use version: 'tags/v0.39.2', see in .github) and update them as well - dynamic_env_pytest_tests/tests/session_token/test_static_session_token_container.py

As stated in #718 

> We have several session token tests failing after https://github.com/nspcc-dev/neofs-node/pull/2731, the node is fine, tests need to be adjusted.
>
> https://http.fs.neo.org/HXSaMJXk2g8C14ht8HSi7BBaiYZ1HeWh2xnWPGQCg4H6/1230-1706624712/index.html#suites/0598eb478b736df2d02c5aa8b49c27f4

**Comment by MaxGelbakhiani** [comment]
And current changes should change all the tests to green after [node PR 2741 ](https://github.com/nspcc-dev/neofs-node/pull/2741) is merged. For now, we'll have 2 yellow tests in this suite.

**Comment by roman-khimov** [comment]
Version here can be updated to 0.40.0, we'll deal with test/node sync after we're quit using dev-env.

**Review by evgeniiz321 — APPROVED** [review]

File: pytest_tests/testsuites/session_token/test_static_session_token_container.py [hunk 1]
```diff
@@ -306,7 +306,6 @@ def test_static_session_token_container_set_eacl(
                 session_token=static_sessions[ContainerVerb.SETEACL],
             )
             wait_for_cache_expired()
-
```
Comment (roman-khimov) [inline pytest_tests/testsuites/session_token/test_static_session_token_container.py:309]: Whitespace change mixed with others.

Comment (MaxGelbakhiani) [reply pytest_tests/testsuites/session_token/test_static_session_token_container.py]: fixed

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**